jupyter_notebooks/pylibCZIrw/pylibCZIrw_4_0_0.ipynb

```diff
@@ -0,0 +1,1276 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ff1c67a3",
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Install pylibCZIrw and dependencies\n",
+    "! pip install --upgrade pip\n",
+    "! pip install \"pylibCZIrw>=4.0\" \"cztile>=0.0,<1.0\" matplotlib tqdm scikit-image pooch requests"
```

Comment:
Can we restrict the requirements to alsway coose 4 as a major version? E.g. pylibCZIrw~=4.0

Reply:
done